src/Exception/AccessDeniedHttpException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException as BaseAccessDeniedHttpException;
+
+/**
+ * This exception is thrown when the access to a resource has been denied.
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class AccessDeniedHttpException extends BaseAccessDeniedHttpException
```

Comment:
Thought about that one too but have not yet implemented it as we might want to export the rendering into an sub request or the like.
Implementing it also in here will add the same amount to the PR.
Downside is also, that the 403 is throwing 404 exceptions internally and even redirect exceptions therefore implementing and testing would imply extracting of the rendering into an own sub handler.
I also thought about pushing parameters like the root page into the request attributes but doing so is beyond the basic functionality of this PR.